src/lib/ursadb.py

```diff
@@ -25,9 +25,11 @@ def query(self, query: str, taint: str) -> Json:
         start = time.clock()
         if taint:
             taint = taint.replace('"', '\\"')
-            query = 'select with taints ["{}"] {};'.format(taint, query)
+            query = 'select with taints ["{}"] into iterator {};'.format(
```

Comment:
Any reason for using formats instead of fstrings like in rest of the code?

Reply:
No, it was just like this before my changes. Fixed!